README.md

```diff
@@ -60,37 +60,27 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Hello World <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+> this is my Hello World project i have created using HTML, CSS, linters
 
 ## 🛠 Built With <a name="built-with"></a>
 
+1: Html
+2: CSS
+3: linters
+
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
 
```

Comment:
[OPTIONAL] 
- [ ] please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1: